spec/spec_helper.rb

```diff
@@ -22,6 +22,9 @@
 
 RSpec.configure do |config|
   config.order = "random"
+
+  # Don't run integration tests by default
+  # config.filter_run_excluding integration: true
```

Comment:
Should this line be commented out?

Reply:
I think these tests should be run separately maybe so yes.